src/offset/local/windows_sys/file_time.rs

```diff
@@ -0,0 +1,50 @@
+use std::{io::Error, result::Result};
+
+use super::WinSystemTime;
+use crate::windows_sys_call;
+
+use winapi::shared::minwindef::*;
+use winapi::um::timezoneapi::*;
+
+const HECTONANOSECS_IN_SEC: i64 = 10_000_000;
+const HECTONANOSEC_TO_UNIX_EPOCH: i64 = 11_644_473_600 * HECTONANOSECS_IN_SEC;
+
+pub(crate) struct WinFileTime {
+    inner: FILETIME,
+}
+
+impl WinFileTime {
```

Comment:
I don't think we should keep the `WinFileTime` as a separate type internally. As far as I can tell the only use we have for it is to implement `as_unix_seconds()`? So IMO it would be good to implement `as_unix_seconds()` on `WinSystemTime` directly instead (inlining all the necessary operations for creating a `FILETIME` etc).